03-transactions.md

```diff
@@ -1566,6 +1692,228 @@ All of them use the following secrets (and thus the derived points):
     # => 0xd09ffff62ddb2297ab000cc85bcb4283fdeb6aa052affbc9dddcf33b61078110
     revocationprivkey: 0xd09ffff62ddb2297ab000cc85bcb4283fdeb6aa052affbc9dddcf33b61078110
 
+# Appendix F: Commitment and HTLC Transaction Test Vectors (anchors)
```

Comment:
nit: add to table of contents at the beginning of the file?

Reply:
added